framework/cdv-gradle-config-defaults.json

```
@@ -6,7 +6,7 @@
     "MIN_BUILD_TOOLS_VERSION": "32.0.0",
     "AGP_VERSION": "7.2.1",
     "KOTLIN_VERSION": "1.5.21",
-    "ANDROIDX_APP_COMPAT_VERSION": "1.4.2",
+    "ANDROIDX_APP_COMPAT_VERSION": "1.5.1",
```

```suggestion
    "ANDROIDX_APP_COMPAT_VERSION": "1.6.1",
```
1.6.1 is now available